### DISCUSSION TITLE
Tag Support

### DISCUSSION THREAD

**Discussion by JosephTLyons** [community/community#3673] [Mobile]
I would love to be able to see the tags on commits on GitHub Mobile.  It might also be nice to have a list of tags somewhere for the repo that you could click and to scroll the history view down to that commit.

**Comment by brianlovin** [comment]
@JosephTLyons thanks for the feedback. Could you share more about this use case, I'm not sure I'm totally following:

1. You want to see tag names on commits in the overall commits list for a repo?
2. We don't support showing all tags, but we do support releases in the apps now (if you're on the latest version) - does this get you close enough to what you want?

**Comment by JosephTLyons** [comment]
Hello,

Point #1 is correct.  I really enjoy being able to see tags in the commit history on GitHub Desktop.  Being able to see that information in the history section is extremely convenient to see what exact commits belong to what releases.  The new releases section is a nice addition, but not all organizations use GitHub Releases, such as the one I work for, but we do use tags like crazy.  Being able to see tags in the commit history, on the commits, would just be a really nice quality of life addition.
